options/locale/locale_en-US.ini

```diff
@@ -2183,7 +2183,7 @@ settings.protect_merge_whitelist_committers_desc = Allow only whitelisted users
 settings.protect_merge_whitelist_users = Whitelisted users for merging:
 settings.protect_merge_whitelist_teams = Whitelisted teams for merging:
 settings.protect_check_status_contexts = Enable Status Check
-settings.protect_check_status_contexts_desc = Require status checks to pass before merging. Choose which status checks must pass before branches can be merged into a branch that matches this rule. When enabled, commits must first be pushed to another branch, then merged or pushed directly to a branch that matches this rule after status checks have passed. If no contexts are selected, the last commit must be successful regardless of context.
+settings.protect_check_status_contexts_desc = "Require status checks to pass before merging. Enter patterns to specify which status checks must pass before branches can be merged into a branch that matches this rule. Multiple patterns can be separated using semicolon (';'). When enabled, commits must first be pushed to another branch, then merged or pushed directly to a branch that matches this rule after status checks have passed. If no contexts are selected, the last commit must be successful regardless of context."
```

Comment:
I would suggest to put a `textarea` and new new line to separate items.

Reply:
Good idea. I'll improve it.